go/cmd/vtbackup/vtbackup.go

```
@@ -304,15 +302,8 @@ func takeBackup(ctx context.Context, topoServer *topo.Server, backupStorage back
 		if err := mysqld.ResetReplication(ctx); err != nil {
 			return fmt.Errorf("can't reset replication: %v", err)
 		}
-		cmds := mysqlctl.CreateReparentJournal()
-		cmds = append(cmds, fmt.Sprintf("CREATE DATABASE IF NOT EXISTS %s", sqlescape.EscapeID(dbName)))
-		if err := mysqld.ExecuteSuperQueryList(ctx, cmds); err != nil {
-			return fmt.Errorf("can't initialize database: %v", err)
-		}
-
-		// Execute Alter commands on reparent_journal and ignore errors
-		cmds = mysqlctl.AlterReparentJournal()
-		_ = mysqld.ExecuteSuperQueryList(ctx, cmds)
+		cmd := mysqlctl.GenerateInitialBinLogEntry()
+		_ = mysqld.ExecuteSuperQueryList(ctx, []string{cmd})
```

So much cleaner. 💯

---

Why is it safe to ignore any errors?